File: stress/t/opt_spill_loop.c
#include <stdio.h>
#include <stdint.h>
int main(void){
  int64_t b0=1,b1=2,b2=3,b3=4,b4=5,b5=6,b6=7,b7=8,b8=9,b9=10,b10=11,b11=12,
    b12=13,b13=14,b14=15,b15=16,b16=17,b17=18,b18=19,b19=20,b20=21,b21=22,b22=23,b23=24,
    i=0,acc=0;
  for(i=0;i<50;i=i+1){
    acc=acc+b0+b1+b2+b3+b4+b5+b6+b7+b8+b9+b10+b11;
    acc=acc+b12+b13+b14+b15+b16+b17+b18+b19+b20+b21+b22+b23;
    acc=(int64_t)((uint64_t)acc+(uint64_t)i+(uint64_t)((int64_t)((uint64_t)b0*(uint64_t)b23)));
  }
  printf("acc="); printf("%ld",(long)acc); printf("\n");
  printf("b0="); printf("%ld",(long)b0); printf("\n");
  printf("b23="); printf("%ld",(long)b23); printf("\n");
  return 0;
}

File: stress/t/opt_spill_loop.f
      PROGRAM OSPLOOP
      INTEGER*8 B0,B1,B2,B3,B4,B5,B6,B7,B8,B9
      INTEGER*8 B10,B11,B12,B13,B14,B15,B16,B17,B18,B19
      INTEGER*8 B20,B21,B22,B23,I,ACC
      B0=1
      B1=2
      B2=3
      B3=4
      B4=5
      B5=6
      B6=7
      B7=8
      B8=9
      B9=10
      B10=11
      B11=12
      B12=13
      B13=14
      B14=15
      B15=16
      B16=17
      B17=18
      B18=19
      B19=20
      B20=21
      B21=22
      B22=23
      B23=24
      ACC=0
      I=0
   10 IF (I.GE.50) GOTO 20
      ACC=ACC+B0+B1+B2+B3+B4+B5+B6+B7+B8+B9+B10+B11
      ACC=ACC+B12+B13+B14+B15+B16+B17+B18+B19+B20+B21+B22+B23
      ACC=ACC+I+B0*B23
      I=I+1
      GOTO 10
   20 CONTINUE
      WRITE(*,'(A,I0)') 'acc=',ACC
      WRITE(*,'(A,I0)') 'b0=',B0
      WRITE(*,'(A,I0)') 'b23=',B23
      END
